styx-api-service/src/main/java/com/spotify/styx/api/WorkflowResource.java

```diff
@@ -103,8 +119,9 @@ public WorkflowResource(Storage storage, WorkflowValidator workflowValidator,
             Middlewares.<Workflow>authed(requestAuthenticator).and(json()), "POST", BASE + "/<cid>",
             rc -> ac -> createOrUpdateWorkflow(arg("cid", rc), rc, ac)),
         Route.with(
-            Middlewares.<ByteString>authed(requestAuthenticator).and(json()), "DELETE", BASE + "/<cid>/<wfid>",
-            rc -> ac -> deleteWorkflow(arg("cid", rc),arg("wfid", rc), ac)),
+            Middlewares.<ByteString>authed(requestAuthenticator).and(json()), "DELETE",
+            BASE + "/<cid>/<wfid>",
+            rc -> ac -> deleteWorkflow(arg("cid", rc), arg("wfid", rc), ac)),
```

Comment:
try to avoid refactoring in commits that add/change the logic. It makes it more difficult to review the actual changes